src/dialects/abstract/query-generator.js

```diff
@@ -83,37 +74,6 @@ export class AbstractQueryGenerator {
     throw new Error(`Databases are not supported in ${this.dialect.name}.`);
   }
 
-  extractTableDetails(tableName, options) {
-    options = options || {};
-    tableName = tableName || {};
-
-    return {
-      schema: tableName.schema || options.schema || this.options.schema || 'public',
-      tableName: _.isPlainObject(tableName) ? tableName.tableName : tableName,
-      delimiter: tableName.delimiter || options.delimiter || '.',
-    };
-  }
-
-  // TODO: always return an object, instead of sometimes an object, sometimes a string
-  addSchema(param) {
```

Comment:
this has been permanently removed. It had a weird behavior and was mostly used when schemas were passed separately from the table name, which is something we're removing (from queryInterface/queryGenerator).